integration/package-operator/objecttemplate_test.go

```diff
@@ -44,7 +44,7 @@ func TestObjectTemplate_creationDeletion_packages(t *testing.T) {
 
 	kubernetesKey := "kubernetes"
 	kubernetesPath := ".environment.kubernetes.version"
-	kubernetesValue := "v1.27.1"
+	kubernetesValue := "v1.27.3"
```

Comment:
Sooo... I don't exactly remember, but did we find out why this change needs to happen?

Reply:
because the int test has a specific k8s version hard coded and fixing this is not part of the PR :|

Reply:
Issue created

Reply:
Alrighty... thankersens!